dev-python/click/click-6.7-r2.ebuild

```diff
@@ -1,9 +1,9 @@
-# Copyright 1999-2020 Gentoo Authors
```

Comment:
6.7-r2 is already partly stable and should not be changed without revbump
And there is also 7.1.2-r2 ...